axolotl-web/src/pages/ChatList.vue

```diff
@@ -23,26 +23,26 @@
       <div v-for="chat in chatList" :key="chat.id" class="row">
         <!-- chat entry -->
         <div
-          :id="chat.ID"
```

Comment:
Do you have an example for how this could be done? The changes here are happening because in rust a variable shouldn't start with an uppercase letter.